projects/agent-framework-javascript/agent-framework-javascript.md

```diff
@@ -0,0 +1,97 @@
+
+![afj-high-level-arch.png](./afj-high-level-arch.png)
+
+> **NOTE<sup>1<sup>:** In addition to the [Aries Framework JavaScript repository](https://github.com/hyperledger/aries-framework-javascript), we propose moving the [Aries Framework JavaScript Extensions](https://github.com/hyperledger/aries-framework-javascript-ext) repository too, due to its tight relationship with the framework.
+>
+> **NOTE<sup>2<sup>:** Another Hyperledger project that is tightly related to the framework is [Aries Mobile Agent React Native (a.k.a. Bifold)](https://github.com/hyperledger/aries-mobile-agent-react-native). This project is maintained by a different set of people and is therefore not included in this proposal.
+
+# Alignment with the OpenWallet Foundation Mission
+
+The objective is to promote interoperability by supporting numerous standards, as was outlined earlier. As a Hyperledger project, all components have been open source from the beginning.
+
+Maintainers will persist in collaborating with Standards Development Organizations (SDOs) across ecosystems, as done previously.
+
+# Code of Conduct
+
+The project currently follows the Hyperledger Code of Conduct as is the case for all Hyperledger projects: [Hyperledger Code of Conduct](https://wiki.hyperledger.org/display/HYP/Hyperledger+Code+of+Conduct). Hyperledger being part of the Linux Foundation too, it may be assumed its Code of Conduct is in line with the foundation's policies.
+
+# TAC Sponsor
+
+Tracy Kuhrt
+
+# Project License
+
+The project has an Apache 2.0 license: [Aries Framework JavaScript - License)](https://github.com/hyperledger/aries-framework-javascript%23license)
+
+# Source Control
+
+The project is hosted in GitHub and includes the following repositories:
+
+- The framework [Aries Framework JavaScript)](https://github.com/hyperledger/aries-framework-javascript%23license)
+- The framework's extensions: [Aries Framework JavaScript Extensions](https://github.com/hyperledger/aries-framework-javascript-ext%23license)
+
+# Issue Tracker
+
+The issues are tracked using GitHub's Issues feature in the corresponding repository. For issues that span across multiple repositories, GitHub's Projects feature may be used.
+
+# External Dependencies
+
+Please see [Aries Framework JavaScript - Dependency Licenses](./dependency-licensing-overview.md)
+
+# Release Methodology
+
+All proposed repositories have continuous deployment/delivery pipelines built using [GitHub Actions](https://github.com/features/actions). The individual packages follow the [_semantic versioning_](https://semver.org/) method, ensuring consistency and safety. Whenever a pull request is merged into the `main` branch that doesn't cause a _major_, _minor_ or _patch_ version increase, an _alpha_ version is released.
```

Comment:
```suggestion
All proposed repositories have continuous deployment/delivery pipelines built using [GitHub Actions](https://github.com/features/actions). The individual packages follow the [_semantic versioning_](https://semver.org/) method, ensuring consistency and safety. Whenever a pull request is merged into the `main` branch an _alpha_ version is released.
```

Reply:
Will the transfer to OWF affect namespaces and built/released libraries? Should we expect downstream projects to be affected by this move? If yes do we have a plan on how to handle it?

Reply:
@skounis, good question. Most packages are currently scoped under the `aries-framework` namespace (e.g. `@aries-framework/core`). Aries being part of the Hyperledger brand, releases following the transfer will have to be released under a different namespace.

To my knowledge, the way to handle this in NPM is by [deprecating](https://docs.npmjs.com/cli/v9/commands/npm-deprecate) the packages under the old namespace with a message that refers developers to the new namespace/package name. If there are other ways of handling this, I'm all ears!

Reply:
@karimStekelenburg you didn't apply my suggestion, but marked it as resolved. Every pull request that is merged to main, will have an alpha release. The deployment is then separate, and can be for a commit where there's already an alpha release for. 

So keeping the "that doesn't cause a _major_, _minor_ or _patch_ version increase," part in is incorrect